python/README.md

```diff
@@ -0,0 +1,64 @@
+# Installation
+
+To use the `aida_interchange` package within your project you must first install it using the instructions described in the [Install setup.py](install-setup.py) section. It is recommended that you install `aida_interchange` into a virtual environment for development purposes. To install `aida_interchange`  within your virtual environment first activate your virtual environment and run the install command below. See [Python Development Setup](python-development-setup) for more details on creating a virtual environment.
+
+# Install setup.py
+
+To install `aida_interchange`, run the following command within the `python` directory:
+
+```bash
+python3 setup.py install
+```
+The `aida_interchange` modules can now be imported into your project.
+
+# Python Development Setup
+
+Setting up a virtual environment for Python development is the recommended best way to mitigate problems between multiple systems working on this project.
+
+## Install virtualenv
+
+```bash
+$ cd ~
+$ mkdir .virtualenvs
+$ sudo pip install virtualenv
+```
+Verify virtualenv is installed
+
+```bash
+$ which virtualenv
+```
+
+## Create virtual environment
+
+Navigate inside the `python` directory and run the following
+
+```bash
+$ virtualenv -p python3 ~/.virtualenvs/aida-interchange-format
+$ source ./virtualenvs/aida-interchange-format/bin/activate
+$ pip install -r requirements.txt
+```	
+
+## Activate virtual environment
+
+Your virtual environment will be automatically activated after running the virtualenv command. If you need to re-activate your virtual environment you can run the following command while in the root python folder of the project
+
+```bash
+$ source ./virtualenvs/aida-interchange-format/bin/activate
+```
+
+## Deactivate virtual environment
+
+To deactivate your current virtual environment run the following command.
+
+```bash
+$ deactivate
+```
+
+# Documentation
+
+The python project uses [Sphinx](http://www.sphinx-doc.org/en/master/) for generating documentation. To generate the documentation, navigate to the `python/docs` directory and run the `update_documentation.sh` script.
+
+```bash
+$ ./update_documentation.sh
```

Comment:
Also, I'm getting an error when running the update_documentation script:
```
WARNING: autodoc: failed to import module 'setup'; the following exception was raised:
Traceback (most recent call last):
  File "/home/HQ/dgemoets/.virtualenvs/aida-interchange-format/lib/python3.6/site-packages/sphinx/ext/autodoc/importer.py", line 32, in import_module
    return importlib.import_module(modname)
  File "/home/HQ/dgemoets/.virtualenvs/aida-interchange-format/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/HQ/dgemoets/Documents/git/test/996/python/setup.py", line 8, in <module>
    long_description=open('README.md').read(),
FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
```

Reply:
I assumed this would be a separate ticket. Since it looks like there is an error generating the documentation it may not be a bad idea to just do it in this ticket and I will work out all the issues. @craigwarsaw i'll let you decide if you want to include this change.

Reply:
I have fixed the bug when generating the documentation. I'll wait on actually moving the `update_documentation.sh` script into the `scripts/docs` folder.

Reply:
> As we've discussed, we might end up moving this to `$VERDI/scripts/docs`.

I want to make sure I'm following...  The documentation generation script we discussed moving was for VERDI Python API.  Did we discuss moving the AIF Python API documentation generation script?  In any case, the generation of AIF documentation should stay in the AIF repo.

Reply:
Indeed, I was mixing up my VERDI and AIF `update_documentation.sh` scripts.  Do you want the AIF script to be in `./python/docs` where it is now, or in `./scripts/` or `./scripts/docs/`?  The latter seems more VERDI-like, but I am fine with leaving it where it is now.

Reply:
I'd leave it where it is as long as it is clear in the README how to use it.  There is no Java version of this script.  The Java README just shows the command you need to generate the docs.